src/styles/main.scss

```
@@ -1,3 +1,73 @@
+$week: (1 'mon', 2 'tue', 3 'wed', 4 'thu', 5 'fri', 6 'sat', 7 'sun');
+$width-day: 100px;
+$height-day: 100px;
```

No need to duplicate variables with same values
```suggestion
$day-size: 100px;
```